eng/Versions.props

```diff
@@ -17,7 +17,7 @@
     <MicrosoftAspNetCoreAppPackageVersion>$(MicrosoftAspNetCoreAllPackageVersion)</MicrosoftAspNetCoreAppPackageVersion>
     <MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>2.2.0</MicrosoftAspNetCoreDeveloperCertificatesXPlatPackageVersion>
     <MicrosoftNETCoreDotNetHostResolverPackageVersion>2.2.1</MicrosoftNETCoreDotNetHostResolverPackageVersion>
-    <MicrosoftBuildPackageVersion>16.0.0-preview.360</MicrosoftBuildPackageVersion>
+    <MicrosoftBuildPackageVersion>16.1.0-preview.19</MicrosoftBuildPackageVersion>
```

Comment:
I thought we weren't taking 16.1 deps yet (@livarcocc?). This should work at `16.0.443` (I think).

Reply:
I guess this could be 16.0.443 (it's a "reference only" dependency), but we already have 16.1 in master of toolset, that's where I got this version from.

Reply:
I have no problem with plowing forward, as long as we're doing it on purpose, so :shipit: